.github/workflows/ci.yml

```diff
@@ -16,6 +16,41 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ steps.get-versions.outputs.node }}
+          cache: yarn
```

Comment:
blocking (proposal): I think we should hold off on using yarn cache, at least for now… While it's a nice optimization, it makes builds non-idempotent wrt its dependencies and forcing a refetch of the cached version is difficult at best.

Reply:
what would be the issue with using the yarn cache? it speeds up package retrieval
see https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies
we use this all the time locally

Reply:
Since we use a lockfile, I don't see an issue with caching.